spec/features/root_page_spec.rb

```diff
@@ -4,8 +4,12 @@
 
 RSpec.describe "User on root path", type: :feature do
   it "sees a list of available chatrooms" do
-    chatroom1 = Room.create!(name: "room 1")
-    chatroom2 = Room.create!(name: "room 2")
+    chatroom1 = Room.new(name: "room 1")
```

Comment:
`room 1`, ahead of @AllPurposeName from `room 2`. Damn straight. Nice spec right here.